estate/models/estate_property.py

```diff
@@ -0,0 +1,98 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from dateutil.relativedelta import relativedelta
```

Comment:
nitpick: import order https://www.odoo.com/documentation/19.0/contributing/development/coding_guidelines.html#imports

```suggestion
from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
from odoo.exceptions import UserError
```

Reply:
thx! fixed.